### DISCUSSION TITLE
Documentation unreachable ?

### DISCUSSION THREAD

**Discussion by eleon-sqli** [js-fns/date-fns#2158] [General]
Hi all. When I click the API documentation link on a Chrome browser, a blank page is displayed telling : "Failed to load package list!"
Have you seen this problem ?

**Comment by leshakoss** [comment]
Hello! Could you please post the exact link?

**Reply by eleon-sqli** [reply]
https://date-fns.org/docs/Getting-Started

Perhaps an enabled extension ? I continue searching.

**Reply by leshakoss** [reply]
Are there any errors or other messages in the browser console?

**Reply by eleon-sqli** [reply]
Very strange. I've disabled all my Chrome extensions and refresh the doc page, no more problem. Then re-enabled all extensions and no problem anymore too.
We can close this discussion... thank you for your help @leshakoss :)

**Reply by leshakoss** [reply]
No problem!

**Comment by mschadegg** [comment]
I have same problem in Chrome, but Works finde in Edge. 
I have tried to deactivate every extension, without success.

I get this message in the console:
```
@firebase/firestore: Firestore (8.2.2): Could not reach Cloud Firestore backend. Backend didn't respond within 10 seconds.
This typically indicates that your device does not have a healthy Internet connection at the moment. The client will operate in offline mode until it is able to successfully connect to the backend.

```
